viam-cartographer/src/carto_facade/carto_facade.h

```diff
@@ -2,9 +2,13 @@
 #ifndef CARTO_FACADE_H
 #define CARTO_FACADE_H
 
+#ifdef __cplusplus
```

Comment:
@nicksanford I actually think these are C++ libraries, so they need to be within an #ifdef __cplusplus